cpu/stm32/stmclk/stmclk_l4wb.c

```diff
@@ -425,5 +584,36 @@ void stmclk_init_sysclk(void)
         }
     }
 
+    /* Enable the LSE is required for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSE)) {
+        stmclk_dbp_unlock();
+        RCC->BDCR |= RCC_BDCR_LSEON;
+        while (!(RCC->BDCR & RCC_BDCR_LSERDY)) {}
+        stmclk_dbp_lock();
+    }
+
+    /* Enable the LSI is required for MCO */
+    if (IS_ACTIVE(CLOCK_ENABLE_LSI)) {
+#if defined(CPU_FAM_STM32WB)
+        RCC->CSR |= RCC_CSR_LSI1ON;
+        while (!(RCC->CSR & RCC_CSR_LSI1RDY)) {}
+#else
+        RCC->CSR |= RCC_CSR_LSION;
+        while (!(RCC->CSR & RCC_CSR_LSIRDY)) {}
+#endif
+    }
```

Comment:
Is there no case for enabling `LSE` and `LSI` if not for `MSO`?

Reply:
RTT and RTC can use LSE or LSI, depending on the availability of an LSE on the board. So we could indeed be smarter, and:
-  enable LSE when RTT/RTC is used and the board provides an LSE or when it's used for MCO
-  enable LSI when RTT/RTC is used but the board has no LSE or when it's used for MCO

That would also need to adapt the RTT/RTC peripherals implementations. The problem is that they are common so this can only be done once all families are adapted. Since MCO is disabled by default, I think it's ok to wait until this is done.

Reply:
Ok, fine with that can you simply adapt the comment to reflect this?

Reply:
Same for the other PR's

Reply:
Comments updated, let me know if it's ok for you.